### DISCUSSION TITLE
Mapview show as globe in react native

### DISCUSSION THREAD

**Discussion by ChandanBeraTechno** [react-native-maps/react-native-maps#4400] [Q&A]
### Summary

What is the solution to implement 3d globe in react native (android/iOS).

### Reproducible sample code

```TSX
None.
```

### Steps to reproduce

None.

### Expected result

![Screenshot 2022-08-16 at 5 14 47 PM](https://user-images.githubusercontent.com/108820898/184874038-4cde9e26-ab24-4bea-a47c-6dd8cc648782.png)

### React Native Maps Version

1.1.0

### What platforms are you seeing the problem on?

Android, iOS (Google Maps)

### React Native Version

0.69

### What version of Expo are you using?

Not using Expo

### Device(s)

iPhone, android

**Comment by aidant86** [comment]
React Native Maps is made for a "flat" map, it doesn't support a globe view.

**Comment by ChandanBeraTechno** [comment]
Is there any other solution for globe view?

**Reply by aidant86** [reply]
Check out [this video](https://www.youtube.com/watch?v=HxHRYEx5hNo). Haven't tested this so I can't say whether or not it will work. I believe this method will only work for iOS, not android.

**Reply by ChandanBeraTechno** [reply]
Thanks for sharing your valuable feedback. why android globe part not working?

**Reply by aidant86** [reply]
Google Maps doesn't have a globe view but Apple Maps does, so **iOS has native support for this**. On Android, Google's globe view is available through Google Earth which is _not_ packaged with Google Maps/Android Map SDK. In other words, **Android does not have native support for a globe view.**

**Comment by TehleelMir** [comment]
just add this 

`        mapType={isAndroid ? "satellite" : "hybridFlyover"}
`
